doc_test.go

```diff
@@ -99,7 +99,7 @@ func Example_gcd() {
 func Example_memory() {
 	// Create our `Store` context and then compile a module and create an
 	// instance from the compiled module all in one go.
-	wasmtime_store := NewStore(NewEngine())
+	wasmtimeStore := NewStore(NewEngine())
```

Comment:
Probably more idiomatic to rename this to `store` instead. It'd also line it up with the other example funcs.

Reply:
no, it will conflict with the following store func
```
func Example_memory() {
	// Create our `Store` context and then compile a module and create an
	// instance from the compiled module all in one go.
	wasmtimeStore := NewStore(NewEngine())
	wasm, err := Wat2Wasm(`
          (module
            (memory (export "memory") 2 3)

            (func (export "size") (result i32) (memory.size))
            (func (export "load") (param i32) (result i32)
              (i32.load8_s (local.get 0))
            )
            (func (export "store") (param i32 i32)
              (i32.store8 (local.get 0) (local.get 1))
            )

            (data (i32.const 0x1000) "\01\02\03\04")
          )
        `)
	check(err)
	module, err := NewModule(wasmtimeStore, wasm)
	check(err)
	instance, err := NewInstance(wasmtimeStore, module, []*Extern{})
	check(err)

	// Load up our exports from the instance
	memory := instance.GetExport("memory").Memory()
	size := instance.GetExport("size").Func()
	load := instance.GetExport("load").Func()
	store := instance.GetExport("store").Func() // conflict here

	// some helper functions we'll use below
	call32 := func(f *Func, args ...interface{}) int32 {
```